components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -2030,6 +2067,16 @@ void AdsServiceImpl::GetAds(
           std::move(callback), categories));
 }
 
+void AdsServiceImpl::GetAdConversions(
+    const std::string& url,
+    ads::OnGetAdConversionsCallback callback) {
+  base::PostTaskAndReplyWithResult(file_task_runner_.get(), FROM_HERE,
+      base::BindOnce(&GetAdConversionsOnFileTaskRunner, url,
```

Comment:
`FROM_HERE` - What is that?

Reply:
It is a `MACRO` from Chromium to store the PC from the `FROM_HERE` on the stack for crash dumps as defined in `src/base/location`